lighthouse-core/gather/gatherers/html-without-javascript.js

```diff
@@ -39,17 +41,16 @@ class HTMLWithoutJavaScript extends Gatherer {
     // Reset the JS disable.
     options.disableJavaScript = false;
 
-    const driver = options.driver;
-
-    this.artifact = {};
-    return driver.evaluateAsync(`(${getBodyText.toString()}())`)
+    return options.driver.evaluateAsync(`(${getBodyText.toString()}())`)
       .then(result => {
-        this.artifact = result;
+        this.artifact = {
+          value: result
```

Comment:
it's somewhat mixed, depending on the gatherer (like [manifest](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/gather/gatherers/manifest.js), [accessibility](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/gather/gatherers/accessibility.js), [http-redirect](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/gather/gatherers/http-redirect.js), etc).

To me if there's going to be a (possible) `debugString`, it should just always be an object so you can do a direct comparison to check for errors rather than doing some kind of `typeof artifact === 'string'` dance